src/test/java/address/util/TestSerializationClass.java

```diff
@@ -0,0 +1,73 @@
+package address.util;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * A class used to test serialization and deserialization
+ */
+public class TestSerializationClass {
```

Comment:
Let's try to stick to 'one unit test class per each functional class' and name the test class to match the functional class.